core/edit/insert_between.py

```diff
@@ -5,8 +5,8 @@
 
 @mod.action_class
 class module_actions:
-    def insert_between(before: str, after: str):
+    def insert_between(before: str, after: str, middle=""):
```

Comment:
```suggestion
    def insert_between(before: str, after: str, middle: str=""):
```

I'm pretty sure import will fail without the type specified